compiler/package.go

```diff
@@ -113,6 +113,9 @@ func (pi packageImporter) Import(path string) (*types.Package, error) {
 		}
 		return nil, err
 	}
```

Comment:
See Issue #410. For example honnef.co/... is missing.

Richard Musiol <notifications@github.com> ezt írta (időpont: 2016. febr.
24., Sze, 12:43):

> In compiler/package.go
> <https://github.com/gopherjs/gopherjs/pull/409#discussion_r53927082>:
>
> > @@ -113,6 +113,9 @@ func (pi packageImporter) Import(path string) (*types.Package, error) {
> >  		}
> >  		return nil, err
> >  	}
>
> I agree with @shurcooL <https://github.com/shurcooL>. How can we
> reproduce this?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gopherjs/gopherjs/pull/409/files#r53927082>.
>